packages/admin/cms-admin/src/contentScope/ContentScopeIndicator.tsx

```diff
@@ -1,71 +1,76 @@
-import { css } from "@mui/material";
+import { messages } from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { SvgIconProps, useTheme } from "@mui/material";
 import { styled } from "@mui/material/styles";
 import React from "react";
+import { FormattedMessage } from "react-intl";
+
+const bgGrey = "#5A697E";
 
 interface ContentScopeIndicatorProps {
-    variant?: "default" | "toolbar";
     global?: boolean;
+    scopes?: React.ReactNode[];
 }
 
-export const ContentScopeIndicator: React.FunctionComponent<ContentScopeIndicatorProps> = ({ children, variant = "default", global = false }) => {
+export const ContentScopeIndicator = ({ global = false, scopes = [] }: ContentScopeIndicatorProps) => {
+    const theme = useTheme();
+
+    const scopeNodes: React.ReactNode[] = [];
+
+    scopes.forEach((scope, idx, arr) => {
+        const key = scope?.toString();
```

Comment:
I'd rather just use the index instead of string-ifying ReactNodes, which may be rather large

Reply:
Done